consensus/src/simplex/actors/batcher/actor.rs

```diff
@@ -76,6 +77,7 @@ impl<
         // Initialize data structures
         Self {
             participants,
+            attributable_scheme: scheme.is_attributable(),
```

Comment:
I wonder if we should instead just provide a reporter wrapper that takes this into account 🤔